src/tools/scratch/Scratch.vcxproj

```diff
@@ -21,6 +21,14 @@
       <SubSystem>Console</SubSystem>
     </Link>
   </ItemDefinitionGroup>
+
+  <ItemGroup>
+    <!-- subsume fmt, one of our dependencies, into contypes. -->
+    <ProjectReference Include="..\..\dep\fmt\fmt.vcxproj">
+      <Project>{6bae5851-50d5-4934-8d5e-30361a8a40f3}</Project>
+    </ProjectReference>
+  </ItemGroup>

```

Comment:
Technically shouldn't be necessary, but I'll allow it to make the next guy's life easier for prototyping.

Reply:
yea basically. Almost included til/wil too :P